app/src/main/java/com/github/multimatum_team/multimatum/DeadlineNotification.kt

```diff
@@ -0,0 +1,55 @@
+package com.github.multimatum_team.multimatum
+
+import android.app.AlarmManager
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import com.github.multimatum_team.multimatum.model.Deadline
+
+class DeadlineNotification {
+    companion object {
+        private lateinit var alarmManager: AlarmManager
+        private lateinit var pendingIntent: PendingIntent
+        /*
+        Create a notification channel for reminder notifications
+        Creating an existing notification channel with its original values performs no operation,
+        so it's safe to call this code when starting an app.
+        */
```

Comment:
Documentation is written between `/**` and `*/` (notice the double `*`), please use this convention as well (the doc should appear in green in android studio)